clickhouse-jdbc/src/main/java/com/clickhouse/jdbc/ClickHouseDriver.java

```diff
@@ -152,6 +172,8 @@ public ClickHouseConnection connect(String url, Properties info) throws SQLExcep
         if (!acceptsURL(url)) {
             return null;
         }
+        if (!url.toLowerCase().contains("disable-frameworks-detection"))
```

Comment:
not every case passes properties thru `url` - often properties are used. 
I suggest do this check in `com.clickhouse.jdbc.internal.ClickHouseConnectionImpl#ClickHouseConnectionImpl(com.clickhouse.jdbc.internal.ClickHouseJdbcUrlParser.ConnectionInfo)` - this method is called right after URL is parsed for properties so ConnectionInfo contains all of them.